packages/editor/src/components/post-author/combobox.js

```diff
@@ -44,13 +45,14 @@ export default function PostAuthorCombobox() {
 		<ComboboxControl
 			__nextHasNoMarginBottom
 			__next40pxDefaultSize
-			label={ __( 'Author' ) }
+			label={ __( 'Authorz' ) }
```

Comment:
Typo?

Reply:
ha, this was me trying to get my build working and not seeing changes. will revert!